registry/repositories.go

```diff
@@ -145,6 +146,10 @@ func (registry *Registry) tryFallback(ctx context.Context, regChan chan string,
 				registry.useBasicPreAuth()
 
 				regurl = registry.url("/api/projects")
+
+				var harborProjRepo string = "/api/repositories?project_id="
+				ctx = context.WithValue(ctx, "harborProjRepo", harborProjRepo)
```

Comment:
Why did you choose to carry this value in the context instead of adding to the function signature for `streamHarborProjectsPage` / `streamHarborProjectRepos`?

Reply:
Either would have worked sufficiently, but I tried to maintain the original style. The other implementation would work, though I would need to add the parameter to streamHarborProjectsPage() and also streamHarborProjectRepos(). Do you think this way would be better for readability?